subprojects/core-api/src/main/java/org/gradle/process/JavaForkOptions.java

```diff
@@ -225,21 +208,14 @@ public interface JavaForkOptions extends ProcessForkOptions {
      * Since Gradle 5.6, you can configure the port and other Java debug properties via
      * {@link #debugOptions(Action)}.
      *
-     * @return true when debugging is enabled, false to disable.
+     * @return A Property[true] when debugging is enabled, false to disable.
```

Comment:
:x: This doesn't make much sense. How about just deleting this two lines?

Reply:
Removed